block-BHaabn/index.html

```diff
@@ -0,0 +1,81 @@
+<!-- 
+- Create a gallery using flexbox according to the layout shown below.
+
+![Flex Shrink Property](https://raw.githubusercontent.com/suraj122/AC-STYLE-images/master/flexbox/ex-1.png)
+
+- Using CSS resets is necessary.
+- Use semantic tags and keep the nesting and indentation proper.
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flexbox Assignmnet1</title>
+    <link rel="stylesheet" href="style.css"/>
+    <script src="https://kit.fontawesome.com/7fa3d6c3a1.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="header">
+        <div class="container">
+        <a>
+            <img class="logo" src="logo.jfif" alt="logo"/>
+        </a>
+        <nav class="nav">
+            <a href="#">HOME</a>
+            <a href="#">GALLERY</a>
+            <a href="#">ABOUT</a>
+        </nav>
+        </div>
+    </header>
+    <main>
+        <section class="section1">
+            <div class="container gallery">
+            <div class="heading">
+                <h1>GALLERY</h1>
+            </div>
+            </div>
+        </section>
+        <section class="section2">
+            <div class="container">
+                <article class="col">
+                    <img class="gal-img" src="assets/01.png" alt="articleimg1"/>
+                </article>
+                <article class="col">
```

Comment:
The same goes here also, use div tag instead. Pay attention to small, small things also.